src/Illuminate/Http/Concerns/CanBePrecognitive.php

```diff
@@ -28,7 +28,7 @@ public function filterPrecognitiveRules($rules)
      *
      * @return bool
      */
-    public function isAttemptingPrecognition()
+    public function isAttemptingPrecognition(): bool
```

Comment:
```suggestion
    public function isAttemptingPrecognition()
```

Reply:
@timacdonald Thanks for checking the merge.

I thought return-type declarations would enhance the performance although it's generally minimal.